esmvaltool/cmorizers/data/formatters/datasets/noaaglobaltemp.py

```diff
@@ -0,0 +1,79 @@
+"""ESMValTool CMORizer for "NOAAGlobalTemp data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://www.ncei.noaa.gov/products/land-based-station/noaa-global-temp
+
+Last access
+    20220512
+
+Download and processing instructions
+    Download the following files:
+        [SOURCE]/v5/access/gridded/
+          NOAAGlobalTemp_v5.0.0_gridded_s188001_e202203_c20220408T133258.nc
+"""
+
+import copy
+import logging
+import os
+
+import iris
+from cf_units import Unit
+
+from esmvaltool.cmorizers.data import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _extract_variable(short_name, var, version, filename, cfg, in_dir,
+                      out_dir):
+    """Extract variable."""
+    # load data
+    filepath = os.path.join(in_dir, filename)
+    raw_var = var.get('raw', short_name)
+    cube = iris.load_cube(filepath, utils.var_name_constraint(raw_var))
+
+    # fix time units
+    cube.coord('time').convert_units(
+        Unit('days since 1950-1-1 00:00:00', calendar='gregorian'))
+
+    cmor_info = cfg['cmor_table'].get_variable(var['mip'], short_name)
+
+    # Fix coordinates
```

Comment:
```suggestion
    # Fix coordinates
    cube = iris.util.squeeze(cube)
```
to remove the one-point `z` coordinate

Reply:
@remi-kazeroni I tried that but unfortunately it does not solve the error. It seems to change nothing. Maybe you could have a try?